package.json

```diff
@@ -26,10 +26,15 @@
   },
   "homepage": "https://github.com/jasonslyvia/react-lazyload",
   "devDependencies": {
-    "babel": "~5.8.21",
-    "babel-core": "~5.8.21",
+    "babel": "~6.5.2",
+    "babel-cli": "^6.7.7",
+    "babel-core": "~6.7.7",
     "babel-eslint": "^6.0.2",
-    "babel-loader": "~5.3.2",
+    "babel-loader": "~6.2.4",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
```

Comment:
This is what is [recommended by Babel](http://babeljs.io/docs/plugins/transform-decorators/)